cmd/gardener-extension-provider-local/app/app.go

```diff
@@ -93,6 +94,7 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 			WebhookCertDir:             "/tmp/gardener-extensions-cert",
 			MetricsBindAddress:         ":8080",
 			HealthBindAddress:          ":8081",
+			RecoverPanic:               pointer.Bool(true),
```

Comment:
This means that every extension has to pro-actively set it, right? That's unfortunate and different from how it was before. Can we default this, so that we don't need this line?

Reply:
We could hardcode it to true here: https://github.com/gardener/gardener/blob/63618a66cc08801570a8a069ea1c32a7aea11e5d/extensions/pkg/controller/cmd/options.go#L305

Reply:
Sure, if this fixes my concern above, then let's go this way.